modules/colors/tests/colors.test.ts

```diff
@@ -0,0 +1,164 @@
+import request from 'supertest';
+import app from "../../../app" // "../../../../app";
+describe('color api endpoints', () => {
+ 
+  describe('GET /color/', () => {
+    it('requesting with no qty returns a list of 1 color', async () => {
+      const qty = 1;
+      const response = await request(app).get(`/colors/`);
+      expect(response.body.length).toBe(qty);
+    });
+    it('requesting with non-numerical qty returns a list of 100 color', async () => {
+      // I think this is potentially a bug - we probably should be retuning 1 colour, or throwing an error.
```

Comment:
@computamike Can you raise this as a bug. Thanks. We should be doing a check in that util to make sure its a number and defaulting if it is not.

Reply:
have raised #239 for this